### ISSUE TITLE
While "Universal Remote" is installed along side of it, Chopdown will not chop down trees.

### ISSUE THREAD

**Issue by TeetoKaziim** [Shovinus/ChopDownUpdated#12]
Forge: 14.23.4.2707
Universal Remote: 1.12.0.9.10
Chopdown Updated: 1.0.12

Expected behaviour: When breaking the last of the ground level logs from under a tree, the chopdown mod usually would turn the rest of that tree into falling item entities that fall onto the ground. But with universal remote installed, trees sternly glare at us while floating in the air, dissapproving of our ambitions, our programming, and gravity itself.

Basically entire issue is in the title. Cross-posting from Universal Remote's Issue tracker in the interest of harmony and joy.

https://github.com/orbwoi/UniversalRemote/issues/27

Thankyou also for your time and consideration.

**Comment by Shovinus** [comment]
Going to make an educated guess and say the mod overwrites the standard player class, will have a look.

**Comment by Shovinus** [comment]
Yep, he overwrites the standard player class, just add
"clayborn.universalremote.hooks.entity.HookedEntityPlayerMP"
to your configuration for allowedPlayers.
I will release later, I added the ability to see the users player in the showBlockName command

**Comment by TeetoKaziim** [comment]
Yer a hero, thankyou very much :3
